### PR TITLE
Feat/26 api gateway icon

### PR DESCRIPTION
**Pull request by manh-vv**
add a demo of API gateway to show that it works with icon network

### CODE REVIEW

**Comment by manh-vv** [comment]
# Note

In this MR, you can check demux-sync of ICON-Loop

### Enable watcher on ICON loop

Create `.env` file as below

```text
app_enable_eos=false
app_enable_icon=true
```

File: packages/demux-icon/package.json [hunk 1]
```diff
@@ -15,7 +16,7 @@
     "compile": "tsc -p tsconfig.build.json",
     "prepublishOnly": "yarn build",
     "lint": "tslint --project . --fix",
-    "test": "jest -o"
+    "test": "jest"
   },
   "dependencies": {
     "bignumber.js": "^9.0.0",
```
Comment (quocle108) [inline packages/demux-icon/package.json:22]: Should be added logger to dependencies list?

**Review by quocle108 — APPROVED** [review]
Looks good to me. Pls add logger to dependency

File: packages/demux-icon/package.json (hunk 1 above)
Comment (manh-vv) [reply packages/demux-icon/package.json:22]: Thank @quocle108

**Pull request merged**